devito/data/decomposition.py

```diff
@@ -122,10 +122,10 @@ def __repr__(self):
         return 'Decomposition(%s)' % ', '.join(ret)
 
     def __call__(self, *args, rel=True):
-        """Alias for ``self.convert_index``."""
-        return self.convert_index(*args, rel=rel)
+        """Alias for ``self.convert_index_loc``."""
+        return self.convert_index_loc(*args, rel=rel)
 
-    def convert_index(self, *args, rel=True):
+    def convert_index_loc(self, *args, rel=True):
```

Comment:
I honestly prefer to see the "direction" of the conversion reflected in the method name

something like ``convert_glb_to_loc`` or simply ``glb_to_loc`` (and then ``loc_to_glb``)

Reply:
Currently the direction is 'implied' which I think is fine - but what about `idx_glb_to_loc` and `idx_loc_to_glb`?